pyvista/plotting/plotting.py

```diff
@@ -3136,7 +3136,55 @@ def link_views(self, views=0):
             If ``views`` is int, link the views to the given view
             index or if ``views`` is a tuple or a list, link the given
             views cameras.
+        Examples
+        ---------
```

Comment:
Adding one blank line.
```suggestion

        Examples
        ---------
```